vue-components/src/components/VtkLocal.js

```diff
@@ -240,6 +248,26 @@ export default {
         subscribe();
       }
       await update();
+
+      // Interactor listener
+      // console.log("interactorId", interactorId)
+      // observerTags.push([interactorId, sceneManager.addObserver(interactorId, "RenderEvent", (id, eventName) => {
+      //   console.log("interactor changed", id, eventName);
+      // })]);
+
+      // Camera listener
+      for (let i = 0; i < cameraIds.length; i++) {
+        const cid = cameraIds[i];
+        console.log("Listen to camera", cid);
+        observerTags.push([cid, sceneManager.addObserver(cid, "ModifiedEvent", () => {
+          // crash: sceneManager.unRegisterState(cid);
+          // crash: sceneManager.updateStatesFromObjects();
+          const cameraState = sceneManager.getState(cid);
+          console.log(`${cameraState.Position}`);
+          emit("camera", cameraState);
```

Comment:
Fixed the crash in my development branch. The logs of `cameraState.Position` in console shows x,y,z that change as I rotate the view so I think it works.

Can you suggest how to catch this emission of the `cameraState` and apply it in python? I checked the `protocol.py` and `vtklocal.py` just in case, nothing seems obvious.

Reply:
```python
VtkLocal(rw, camera="camera = $event")


@change("camera"):
def on_camera_change(camera,  **_):
    print("camera state available", camera)
```

Reply:
Thanks! also had to do a small change in `LocalView.__init__`

```py
--- a/trame_vtklocal/widgets/vtklocal.py
+++ b/trame_vtklocal/widgets/vtklocal.py
@@ -75,6 +75,7 @@ class LocalView(HtmlElement):
             "updated",
             ("memory_vtk", "memory-vtk"),
             ("memory_arrays", "memory-arrays"),
+            ("camera", "camera"),
         ]
```